workers.go

```diff
@@ -369,23 +371,23 @@ func (n *Node) processReqStoreEvents(eventsIn *events.EventList) (*events.EventL
 
 			// Store request data.
 			if err := n.modules.RequestStore.PutRequest(storeEvent.RequestRef, storeEvent.Data); err != nil {
-				return nil, fmt.Errorf("cannot store request (c%dr%d) data: %w",
+				return nil, fmt.Errorf("cannot store request (c%sr%d) data: %w",
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)

Reply:
ok